test-case.config.yaml

```diff
@@ -28,6 +31,15 @@ sets:
         name: Releasing a ROS 2 package with bloom
       - isMatch: false
         name: Rosbag with ROS1 Bridge
+      # Ignore links which are redirects.
+      - isMatch: false
+        name: Ignition
+        labels:
+          - ignition
+      - isMatch: false
+        name: Custom ROS2 Interfaces.rst
```

Comment:
I've recently fixed this in docs.ros.org to `Custom ROS2 Interfaces`.  But it actually shows a weakness to this approach; if we still have the tutorial with the same name, but we've moved its directory structure, we don't seem to have a way to deal with the duplicates.

I'm going to suggest that for this one, we actually don't do this here.  Instead, we should probably fix https://github.com/ros2/ros2_documentation/blob/rolling/conf.py so that if we are doing redirects, we make sure to exclude them from the sitemap configuration.

```suggestion
```

Reply:
Sounds good. I'll look into updating the conf.py
https://github.com/audrow/yatm/pull/405/commits/092b3b57c18fe7e8ca36e8593db83b767347a0f9

Reply:
I spent some time looking at how to remove redirected urls from sitemap.xml. I didn't see a trivial way to do it and have a proposal here https://github.com/ros2/ros2_documentation/issues/3733